packages/cozy-client/src/models/instance.spec.js

```diff
@@ -23,6 +30,13 @@ const selftHostedInstance = {
     data: {
       attributes: {}
     }
+  },
+  diskUsage: {
+    data: {
+      attributes: {
+        quota: '6000000000000'
```

Comment:
Is it normal that the quota is a string instead of a number here ?

Reply:
Normal idk, but if the question is: is it coherent with stack's response? yes : https://docs.cozy.io/en/cozy-stack/settings/#response

Reply:
OK for me, cc @nono, is it because encoding of large numbers in JSON should be done via strings ?

Reply:
I don't think we have numbers large enough to have precision issues (by a large margin), but I don't see any other explanations, and it was made a long time ago by people that are no longer in the team, so I assume it was the reason to use a string.